person.rb

```diff
@@ -23,6 +24,11 @@ def correct_name
     @name
   end
 
+  def create_rental(rental)
+    @rentals << rental
+    rental.person = self
+  end
```

Comment:
- [ ] Great job but I think here when creating the rental you should pass the book and date as a paramter to the function then you can generate a new rental. 🚀🔥💯

Something like this
> def add_rental(book, date)
    Rental.new(date,  book, slef)
  end